app-emulation/punes/punes-0.110.ebuild

```diff
@@ -13,14 +13,20 @@ S="${WORKDIR}/puNES-${PV}"
 LICENSE="GPL-2+"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE="X cg ffmpeg"
+IUSE="X cg ffmpeg qt6"
 
 RDEPEND="
-	dev-qt/qtcore:5
-	dev-qt/qtgui:5
-	dev-qt/qtnetwork:5
-	dev-qt/qtsvg:5
-	dev-qt/qtwidgets:5
+	qt6? (
+		dev-qt/qtbase:6[gui,network,opengl,widgets]
+		dev-qt/qtsvg:6
+	)
+	!qt6? (
+		dev-qt/qtcore:5
+		dev-qt/qtgui:5
+		dev-qt/qtnetwork:5
+		dev-qt/qtsvg:5
+		dev-qt/qtwidgets:5
+	)
```

Comment:
USE blocks should come after the other dependencies, so both `qt6` and `!qt6` should be moved below `ffmpeg`.